gems/sorbet/lib/require_everything.rb

```diff
@@ -106,6 +104,41 @@ def self.patch_kernel
 
   private
 
+  def self.rb_file_paths
+    srb_path = __FILE__
+    matcher = %r{^(.*/gems/)sorbet-([.\d]*)/*}
+    match = matcher.match(srb_path)
+
+    unless match
+      raise "Could not match with sorbet at #{srb_path}"
+    end
+
+    gem_path = match[1]
+    version_number = match[2]
+
+    glob = gem_path + "/sorbet-static-#{version_number}*/libexec/sorbet"
+    unless Dir[glob].any?
+      raise "Could not find sorbet-static executable for sorbet at #{srb_path}"
+    end
```

Comment:
Is this entire stanza with matching and file paths etc. just to fork out to `srb tc`? In that case, we already have other places where we do this:

https://github.com/sorbet/sorbet/blob/3aef51ccf0a857fab565d1b3e2566f167c9b64fe/gems/sorbet/lib/todo-rbi.rb#L18-L28

https://github.com/sorbet/sorbet/blob/3aef51ccf0a857fab565d1b3e2566f167c9b64fe/gems/sorbet/lib/suggest-typed.rb#L42-L45

https://github.com/sorbet/sorbet/blob/3aef51ccf0a857fab565d1b3e2566f167c9b64fe/gems/sorbet/lib/hidden-definition-finder.rb#L104-L114

https://github.com/sorbet/sorbet/blob/3aef51ccf0a857fab565d1b3e2566f167c9b64fe/gems/sorbet/lib/hidden-definition-finder.rb#L120-L142

Reply:
Ah, that's much cleaner. I didn't realize `srb tc` was exactly equivalent to calling the executable. Implemented in https://github.com/sorbet/sorbet/pull/1351/commits/2e7a56a8d5c00c6d1c49754d8259adbe0e296a6b